### DISCUSSION TITLE
Pause 30 minutes (alarm clock)

### DISCUSSION THREAD

**Discussion by purpleturtle1988** [basnijholt/adaptive-lighting#434] [Q&A] [answered]
I have installed AM-Droid alarm clock on my Android smartphone, it wakes me up gently with light. I would like to disable Adaptive Lightning for 30 minutes. Is there a service for this?

**Comment by foulk466** [comment, marked as answer]
You can disable it though the switch mentioned in the documentation here:
https://github.com/basnijholt/adaptive-lighting

_The integration creates 4 switches (in this example the component's name is _"living_room"):_

_switch.adaptive_lighting_living_room,_ which turns the Adaptive Lighting integration on or off. It has several attributes that show the current light settings._
